build-info-extractor-gradle/src/main/groovy/org/jfrog/gradle/plugin/artifactory/task/DeployTask.java

```diff
@@ -95,6 +95,22 @@ private void prepareAndDeploy() throws IOException {
             }
         }
 
+        // Extract Build Info.
```

Comment:
This method may be becoming longer than needed. Let's try to split it into smaller sections. Maybe we can take this section out into another method.